cmd/rest/main.go

```diff
@@ -5,13 +5,12 @@ package main
 import (
 	"database/sql"
 	"fmt"
-	"log"
-
 	"github.com/go-devs-ua/octagon/app/repository/pg"
 	rest "github.com/go-devs-ua/octagon/app/transport/http"
 	"github.com/go-devs-ua/octagon/app/usecase"
 	"github.com/go-devs-ua/octagon/cfg"
 	"github.com/go-devs-ua/octagon/migration"
+	"log"
```

Comment:
If we use standard library `"log"`, than it should be placed in the top group of imports